modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_data.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module PdfConstruction
+    module HigherLevelReview::V1
+      class FormData
+        def initialize(higher_level_review)
+          @higher_level_review = higher_level_review
+        end
+
+        delegate :first_name, to: :higher_level_review
+
+        delegate :middle_initial, to: :higher_level_review
+
+        delegate :last_name, to: :higher_level_review
+
+        def first_three_ssn
+          ssn.first(3)
+        end
+
+        def second_two_ssn
+          ssn[3..4]
+        end
+
+        def last_four_ssn
+          ssn.last(4)
+        end
+
+        def birth_month
+          higher_level_review.birth_mm
+        end
+
+        def birth_day
+          higher_level_review.birth_dd
+        end
+
+        def birth_year
+          higher_level_review.birth_yyyy
+        end
+
+        delegate :file_number, to: :higher_level_review
+
+        delegate :service_number, to: :higher_level_review
+
+        delegate :insurance_policy_number, to: :higher_level_review
+
+        def claimant_type(index)
+          index == 4 ? 1 : 'off'
+        end
+
+        def mailing_address_street
+          'USE ADDRESS ON FILE'
+        end
+
+        def mailing_address_unit_number
+          ''
```

Comment:
Are these methods meant to return empty strings?

Reply:
Yep! The purpose of these classes was to abstract away the 'magic responses' that existed for the Higher Level Review PDF form data, and all the mailing address form fields were empty except the street address.

That value is 'USE ADDRESS ON FILE', so once that's updated, we'll have the correct values for the Veteran address.